src/commands/run-tests.yml

```diff
@@ -1,105 +1,113 @@
 description: >
-  Run all tests in a MATLAB project and generate test artifacts. MATLAB includes any files in your
-  project that have a Test label. If your pipeline does not leverage a MATLAB project, then MATLAB
-  includes all tests in the the root of your repository including its subfolders.
+  Run MATLAB and Simulink tests and generate artifacts. By default, the command includes any files
+  in your project that have a `Test` label. If your pipeline does not use a MATLAB project, or if
+  it uses a MATLAB release before R2019a, then the command includes all tests in the root of your
+  repository and in any of its subfolders. The command fails if any of the included tests fails.
```

Comment:
For run-build and run-command, you have included "The command uses the topmost MATLAB version on the system path." For consistency, we need to have the sentence for all three commands or for none. What is your preference?

Reply:
I don't have a strong preference. I imagine in many cases the version of MATLAB we install will be the only one on the path

Reply:
updated